build.gradle.kts

```diff
@@ -127,6 +127,8 @@ subprojects {
 //            "modLocalRuntime"(group = "vazkii.patchouli", name = "Patchouli", version = "$minecraftVersion-$patchouliVersion-${modLoader.uppercase()}")
 //            "modLocalRuntime"(group = "com.lowdragmc.shimmer", name = "Shimmer-$modLoader", version = "$minecraftVersion-$shimmerVersion") { isTransitive = false }
         }
+
+        "modCompileOnly"(group = "me.shedaniel", name = "REIPluginCompatibilities-forge-annotations", version = "8.+")
```

Comment:
should be `compileOnly` and should be moved to `common/build.gradle.kts`
so:
`compileOnly(group = "me.shedaniel", name = "REIPluginCompatibilities-forge-annotations", version = "8.+")`